app/assets/stylesheets/header.scss

```diff
@@ -1,24 +1,69 @@
 // Place all the styles related to the Users controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+$break-xsmall: 481px;
 $break-small: 1024px;
 $break-med: 1288px;
 $grey-darker: hsl(222, 45%, 21%); // box dark blue
 $black-bis: hsl(223.2, 54.35%, 18.04%); // background color
 
 header {
-  padding: 30px 55px 0;
+  padding: 75px 55px 30px;
   margin: 0;
 
   @media screen and (max-width: $break-med) {
-    padding: 30px 85px 30px 85px;
+    padding: 75px 85px 30px 85px;
   }
 
   @media screen and (max-width: $break-small) {
     padding-inline-start: 30px 30px 30px 30px;
```

Comment:
In response to your comment/question, no idea what this is meant to be doing.

Reply:
Should I create a new issue for this? This CSS property is invalid anyways. I think it will be safe to remove it entirely.